unclognearline/src/main/scala/streamcomponents/DeletableSwitch.scala

```diff
@@ -0,0 +1,57 @@
+package streamcomponents
+
+import akka.stream.stage.{AbstractInHandler, AbstractOutHandler, GraphStage, GraphStageLogic}
+import akka.stream.{Attributes, Inlet, Outlet, UniformFanOutShape}
+import models.UnclogStream
+import org.slf4j.LoggerFactory
+
+/**
+  * Checks whether the incoming object contains a VidispineProject object set to deletable.
+  * If so, pushes it to the outYes port, if not pushes it to the outNo port.
+  */
+class DeletableSwitch extends GraphStage[UniformFanOutShape[UnclogStream, UnclogStream]]{
```

Comment:
we have a problem here; thinking about it carefully, we want to match "Deep Archive" if _any_ project is "deepArchive" (as above), but we only want to match "Deletable" if _all_ projects are "deletable".  Please invert the check at line 35 (i.e., push to No immediately and return if the project is _not_ deletable)

Reply:
I wrote this file before you told me we did not need it so I am just going to delete it as it is not used.

Reply:
ok